crates/telemetry/src/exporter.rs

```diff
@@ -14,41 +14,87 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-use std::sync::Arc;
+use std::sync::{Arc, LazyLock};
 
 use arrow::array::RecordBatch;
 use async_trait::async_trait;
 use flight_client::{Credentials, FlightClient};
 use opentelemetry_sdk::metrics::MetricError;
+use secrecy::SecretString;
 
-#[derive(Debug, Clone)]
-pub struct AnonymousTelemetryExporter {
-    flight_client: Option<FlightClient>,
+const ENDPOINT_CONST: &str = "https://telemetry.spiceai.io";
+
+pub static ENDPOINT: LazyLock<Arc<str>> = LazyLock::new(|| {
```

Comment:
Same for this - the telemetry builder should require an endpoint as input. The telemetry.spiceai.io endpoint should only be used for anonymous telemetry